components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js

```diff
@@ -46,6 +46,22 @@ $(document).ready(function() {
 
 });
 
+function getLogStatusIcon(entry) {
+    switch (entry) {
```

Comment:
You don't need to have break after return as returning the function will terminate the flow.